### DISCUSSION TITLE
cannot import name 'cython_msh' from partially initialized module 'simnibs.mesh_tools' (most likely due to a circular import)

### DISCUSSION THREAD

**Discussion by clinerb** [simnibs/simnibs#320] [6. Getting started]
I have downloaded the simnibs package in python, but there are still problems during the import process
![微信截图_20240419162749](https://github.com/simnibs/simnibs/assets/102652655/e565313a-7426-40bc-b704-ae7145032139)
![微信截图2](https://github.com/simnibs/simnibs/assets/102652655/d2630105-68a3-41f4-9bf0-7ca625a86625)
![微信截图2](https://github.com/simnibs/simnibs/assets/102652655/56413293-ef9e-408b-93c6-a8a4f29e6bb2)

**Comment by oulap** [comment]
Did you use the installer, or did you install from the wheel? It also looks like v4.0.0 and not the newest version (4.1).

**Reply by clinerb** [reply]
I have installed the simnibs version(4.1),but I don`t know how to put it in the packages files of python,in another way,I don` know run which program.I uesd the "pip install simnibs" in cmd to download the package,but it failed to work correctly( it showed version4.0)

**Reply by oulap** [reply]
The installer should work, and once the installation is done you can use the simnibs-specific python interpreter called:

`simnibs_python`

You can also grab the environment_<os>.yml file from the github page, create a virtual environment based on that, and then install the simnibs wheel there using pip.
